tests/modules/blockedclient.c

```diff
@@ -85,6 +85,93 @@ int acquire_gil(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     return REDISMODULE_OK;
 }
 
+void *bg_call_worker(void *arg) {
+    typedef struct {
+        RedisModuleString **argv;
+        int argc;
+        RedisModuleBlockedClient *bc;
+    } bg_data;
```

Comment:
@oranagra you probably want to avoid defining this typedef twice.

Reply:
sorry.. i'm lazy.. i'm usually using a trick of instantiating it right from the declaration (instead of a typedef) but i didn't do it this time so there's really no excuse.